build_tools/mako/compile_modules.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Compiles the given model to modules. The targets are expected to be a
+# comma-separated list, e.g., --targets=vmla,vulkan-spirv,dylib-llvm-aot
+#
+# If dylib-llvm-aot target is set, the script assumes ANDROID_NDK env is also
```

Comment:
Looks like it always assumes `ANDROID_NDK` is set. You could put its use inside the case statement to make this comment true.

Given this script is android specific, please name it as such.

Also, this assumes various things about file paths (e.g. `build-android`), so please clarify in the documentation here that this is intended for use by benchmarking automation.

Reply:
done